config/samples/dataplane_v1beta1_openstackdataplanenodeset_bgp.yaml

```diff
@@ -22,16 +22,21 @@ spec:
     - telemetry
   preProvisioned: true
   nodes:
-      edpm-compute-0:
-        hostName: edpm-compute-0
-        ansible:
-          ansibleHost: 192.168.122.100
-          ansibleVars:
-            ctlplane_ip: 192.168.122.100
-            internalapi_ip: 172.17.0.100
-            storage_ip: 172.18.0.100
-            tenant_ip: 172.19.0.100
-            fqdn_internalapi: edpm-compute-0.example.com
+    edpm-compute-0:
+      hostName: edpm-compute-0
+      ansible:
+        ansibleHost: 192.168.122.100
+    networks:
```

Comment:
Wrong indent it seems. `networks` should be a child of `edpm-compute-0`. Same in other samples.

Reply:
Ah true, and I made the same mistake in all of them because I just did 'y' 'p' everywhere.

Fixed them all up now with the extra indent level.